src/Umbraco.Forms.Integrations.Crm.Hubspot/Configuration/HubspotSettings.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+using System.Collections.Specialized;
+
+namespace Umbraco.Forms.Integrations.Crm.Hubspot.Configuration
+{
+    public class HubspotSettings
+    {
+        public HubspotSettings(NameValueCollection appSettings)
+        {
+            var settings = appSettings["Umbraco.Forms.Integrations.Crm.Hubspot.AllowContactUpdate"];
+            AllowContactUpdate = bool.TryParse(settings, out var result)
+                ? result : false;
+        }
+
+        public bool AllowContactUpdate { get; set; }
```

Comment:
```suggestion
        public bool AllowContactUpdate { get; }
```

Looks like you don't need the setter here.